app/src/main/java/fr/free/nrw/commons/category/GridViewAdapter.java

```diff
@@ -98,13 +98,10 @@ public View getView(int position, View convertView, ViewGroup parent) {
      * @param author
      */
     private void setAuthorView(Media item, TextView author) {
-        if (!TextUtils.isEmpty(item.getCreator())) {
-            String uploadedByTemplate = getContext().getString(R.string.image_uploaded_by);
-
-            String uploadedBy = String.format(Locale.getDefault(), uploadedByTemplate, item.getCreator());
-            author.setText(uploadedBy);
+        if (item.getCreator()!="") {
```

Comment:
there are no spaces around operators so this file is not in style with the project.

Also what if `getCreator` returns null, if you `ctrl+click` on `TextUtils.isEmpty` you will see it handles this case